doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should process commands fast.
```

Comment:
Recall from tutorial that NFRs are sort of like constraints, and should be quantified. How do you quantify `process commands fast`?